### PR TITLE
API-2349 added 'agent' as a test user type but disabled

### PR DESCRIPTION
**Pull request by edinhodzic**

### CODE REVIEW

File: app/uk/gov/hmrc/testuser/models/TestUser.scala [hunk 1]
```diff
@@ -30,11 +30,16 @@ case class TestOrganisation(username: String, password: String, saUtr: SaUtr, em
...
   type UserType = Value
   val INDIVIDUAL = Value("INDIVIDUAL")
   val ORGANISATION = Value("ORGANISATION")
+  val AGENT = Value("AGENT")
 
   def from(userType: String) = UserType.values.find(e => e.toString == userType.toUpperCase)
 
-}
\ No newline at end of file
+}
```
Comment (hughfdjackson) [inline app/uk/gov/hmrc/testuser/models/TestUser.scala]: If it's disabled, do we need to add this to the case classes?

File: app/uk/gov/hmrc/testuser/views/userdetails/agent.scala.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    </label>
+    <p data-username>@agent.username</p>
+</div>
+
+<div class="form-field form-field--tall divider--bottom">
+    <label class="label--full-length" id="password-label">
+        <strong>Password</strong>
+    </label>
+    <p data-password>@agent.password</p>
+</div>
```
Comment (hughfdjackson) [inline app/uk/gov/hmrc/testuser/views/userdetails/agent.scala.html]: This should be an unreachable page at this point.

File: app/uk/gov/hmrc/testuser/models/TestUser.scala (hunk 1 above)
Comment (edinhodzic) [inline app/uk/gov/hmrc/testuser/models/TestUser.scala]: True that, one shall remove the surplus case class.

File: app/uk/gov/hmrc/testuser/views/userdetails/agent.scala.html (hunk 2 above)
Comment (edinhodzic) [inline app/uk/gov/hmrc/testuser/views/userdetails/agent.scala.html]: There you go being right again, I'll remove this too.

**Comment by edjannoo** [comment]
Closing this PR as we do not currently plan to offer creation of agents through the frontend

**Pull request closed without merging**